mypy/test/testcheck.py

```diff
@@ -83,14 +80,9 @@
 
 
 class TypeCheckSuite(DataSuite):
-
-    @classmethod
-    def cases(cls) -> List[DataDrivenTestCase]:
-        c = []  # type: List[DataDrivenTestCase]
-        for f in files:
-            c += parse_test_cases(os.path.join(test_data_prefix, f),
-                                  None, test_temp_dir, True)
-        return c
+    files = typecheck_files  # type: List[str]
```

Comment:
All of the variable type annotations for in `DataSuite` subclasses now seem redundant. Can you remove them?